cmd/dp-filter-api/main.go

```diff
@@ -101,79 +113,125 @@ func main() {
 		cfg.EnablePrivateEndpoints,
 		cfg.DownloadServiceURL,
 		cfg.DownloadServiceSecretKey,
+		cfg.ServiceAuthToken,
 		auditor,
+		&hc,
 	)
 
```

Comment:
even if we no longer kill the service on kafka errors, we could log them like so:
```
producer.Channels().LogErrors(ctx, "kafka producer")
auditProducer.Channels().LogErrors(ctx, "kafka audit producer")
```

Reply:
I have added these lines on line 70 / 86 of main.go - just after the initialisation of the producers.